src/Compilers/Core/Portable/PublicAPI.Unshipped.txt

```diff
@@ -51,6 +51,7 @@ Microsoft.CodeAnalysis.IOperation.Syntax.get -> Microsoft.CodeAnalysis.SyntaxNod
 Microsoft.CodeAnalysis.IOperation.Type.get -> Microsoft.CodeAnalysis.ITypeSymbol
 Microsoft.CodeAnalysis.IPropertySymbol.RefKind.get -> Microsoft.CodeAnalysis.RefKind
 Microsoft.CodeAnalysis.IPropertySymbol.ReturnsByRefReadonly.get -> bool
+Microsoft.CodeAnalysis.ITypeSymbol.IsByRefLikeType.get -> bool
```

Comment:
FYI @jaredpar @gafter for public API change

Reply:
What is the word "By" here for? I think it would be more grammatical to call it `IsRefLikeType`.

Reply:
ByRefLike happens to be an existing term in CLR/ECMA for ref-containing types, so it was chosen as the name of the attribute that express that in metadata.  
C# can choose a different word for its APIs. How are we goong to call them in the spec?